python/cudf/cudf/tests/test_mvc.py

```diff
@@ -1,40 +1,19 @@
-# Copyright (c) 2023-2024, NVIDIA CORPORATION.
+# Copyright (c) 2023-2025, NVIDIA CORPORATION.
 import subprocess
 import sys
+from importlib.util import find_spec
 
-import pytest
-from packaging import version
+IS_CUDA_12_PLUS = find_spec("pynvjitlink") is not None
+IS_CUDA_11 = not IS_CUDA_12_PLUS
```

Comment:
Are these used anymore?

Reply:
I also do not see this used outside this file in cuDF Python (and the `find_spec` then can be removed too)

Reply:
```suggestion
```